otter/convergence/steps.py

```diff
@@ -70,12 +102,38 @@ def got_name(random_name):
                 'POST',
                 'servers',
                 data=thaw(server_config),
-                success_pred=has_code(202))
+                success_pred=has_code(202),
+                reauth_codes=(401,))
 
         def report_success(result):
             return StepResult.SUCCESS, []
 
         def report_failure(result):
+            """
```

Comment:
As a clarification - we can make this a public function once we determine how we want logging to happen, etc, and how this will interact with `UpstreamError`.  I am just being lazy because I don't want to write tests for a public function yet, if it needs to be refactored soon.  :)